simulationworkflowschema/general.py

```diff
@@ -476,3 +480,41 @@ class ElectronicStructureOutputs(SimulationWorkflowResults):
         Ref to the Green functions section.
         """,
     )
+
+
+class NMROutputs(SimulationWorkflowResults):
+    """
+    Base class defining the typical output properties of a NMR SinglePoint calculation.
+    """
+
+    magnetic_shielding_nmr = Quantity(
```

Comment:
It seems that it is still not clear to me why the suffix is present in the base class. I understood that the suffix is supplied by the sub_section that uses this class. In case of inheritance, do you think there will be conflicts for these quantities. But again, these conflicts indicate a problematic tree.